addons/sale/report/sale_report_templates.xml

```diff
@@ -166,10 +166,10 @@
 
             <div class="oe_structure"/>
 
-            <p t-field="doc.note" />
-            <p t-if="not is_html_empty(doc.payment_term_id.note)">
+            <div t-field="doc.note" />
+            <div t-if="not is_html_empty(doc.payment_term_id.note)">
                 <span t-field="doc.payment_term_id.note"/>
-            </p>
+            </div>
             <p id="fiscal_position_remark" t-if="doc.fiscal_position_id and not is_html_empty(doc.fiscal_position_id.sudo().note)">
```

Comment:
Might be needed to change the p here as well if I correctly understand the problem :thinking:.